src/pages/informationmodel-details-page/information-model-details-page.jsx

```diff
@@ -247,7 +251,10 @@ const renderStickyMenu = (
     }
   ];
 
-  if (schema) {
+  if (
+    isNewInformationModel({ objectTypes, codeTypes, dataTypes, simpleTypes }) ||
```

Comment:
Er det ikke tydeligere om du legger OR fremfor schema, i stedet for å gjemme det bort på slutten av denne linjen?

Reply:
Ok, jeg kan snu på det